config/crossSections.yaml

```
@@ -0,0 +1,748 @@
+  unc:
+WZTo1L1Nu2Q:
+  crossSec: 10.71
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson
+  unc: 0.01618
+WZTo2L2Q:
+  crossSec: 6.419
+  reference: XSDB WZTo2Q2L_mllmin4p0_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2021-10-11 11:27:06
+  unc: 0.01984
+WWTo2L2Nu:
+  crossSec: 11.09
+  reference: XSDB WWTo2L2Nu_TuneCP5_13TeV-powheg-pythia8 modifiedOn 2021-06-21 10:00:46
+  unc: 0.00704
+WWTo4Q:
+  crossSec: 51.55  #51.723
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson
+  unc: 4.849e-02
+ZZZ:
+  crossSec: 0.01476
+  reference: XSDB ZZZ_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 10:02:03
+  unc: 2.5e-06
+WZZ:
+  crossSec: 0.05709
+  reference: XSDB WZZ_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 10:01:13
+  unc: 6.213e-05
+WWW:
+  crossSec: 0.2158
+  reference: XSDB WWW_4F_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 10:00:48
+  unc: 0.0002479
+WWZ:
+  crossSec: 0.1707
+  reference: XSDB WWZ_4F_TuneCP5_13TeV-amcatnlo-pythia8 modifiedOn 2021-06-21 10:00:50
+  unc: 0.0001757
+ZH_HToBB_ZToLL:
+  crossSec: ( 0.033696 +0.033662 + 0.033632)*0.5824*0.8839
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+ZH_HToBB_ZToQQ:
+  crossSec: ( 0.69911)*0.5824*0.8839
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+ZHToTauTau:
+  crossSec: 0.06272*0.8839
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+WplusHToTauTau:
+  crossSec: 0.8400 * 0.06272
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+WminusHToTauTau:
+  crossSec: 0.5328* 0.06272
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+VBFHToTauTau:
+  crossSec: 3.782*0.06272
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+GluGluHToTauTau:
+  crossSec: 48.58*0.06272
+  reference: =Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+  unc:
+WW:
+  crossSec: 75.95
+  ref: XSDB WW_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 10:00:53
+  unc: 0.1124
+WWTo1L1Nu2Q:
+  crossSec: 51.09
+  reference: XSDB WWTo1L1Nu2Q_4f_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-11-14 11:13:58
+  unc: 0.1353
+WZ:
+  crossSec: 27.59
+  reference: XSDB WZ_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 10:01:15
+  unc: 0.03993
+ggX_HH_bbtautau:
+  crossSec: 1
+  reference:
+  unc:
+VBFX_HH_bbtautau:
+  crossSec: 1
+  reference:
+  unc:
+ggHH_bbTauTau_SM:
+  crossSec: 31.05*0.001 *0.5824*0.06272
+  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
```

```suggestion
  crossSec: 31.05*0.001 *0.5824*0.06272 * 2
  reference: Higgs_XSBR_YR4_update https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
```
- for HH processes values from this twiki https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGHH

---

Why *2 ? I missed this

---

The HH process was taken by https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGHH#Latest_recommendations_for_gluon and the BRs from the YR

---

combinatorics H1->bb & H2->tautau, H1->tautau & H2->bb